06-building-a-program-start-to-finish/using-design-patterns/Output/README.md

```diff
@@ -0,0 +1,159 @@
+### Output 
+#### Design Patterns
+* A design pattern is a general repeatable solution to a commonly occurring problem in software design.
+* Design patterns can speed up the development process by providing tested, proven development paradigms.
+* Reusing design patterns helps to prevent subtle issues that can cause major problems and improves code readability.
+
+#### Design Patterns Used
+### Module Design Pattern
+
+``` 
+/**
```

Comment:
Not particularly sure you are to slam your entire invertedIndex class as part of this output. I figure that based on the design pattern you want to elaborate on, a snippet will suffice. Please review.